src/result.js

```diff
@@ -479,22 +479,25 @@ var ResultList = CreateClass({
     
         var addPercent = (value) => value + "%";
         var addBlank = (value) => value;
+        // For avoid a "別枠バフ2XX%"
+        var addPercentAndSpace = (value) => (locale == "en" ? "" : " ")+ value + "%";
```

Comment:
it's being worse
- the maintenance cost of update code
  - should not have the "locale" dependency code in View (result.js)
  - doing the same twice in clone-code (code which copy-and-pasted)
    we should avoid such copy-and-paste code as possible, when update the code
    we have to search similar code and update them together.
    This is known code design problem, similar cause happened in #321 
    The cause is not just lack of tests, the code design issue.
- I've already mention the solution, unify the format can solve this code also become.
  whatever separator

Reply:
done